Yoast/wordpress-seo
# Change activeKeyword-based logic to openSidebarSections-based logic

## Summary

This PR can be summarized in the following changelog entry:

* Makes sure you can open multiple keyword sections at the same time.

## Relevant technical choices:

* Previously activeKeyword kept track of the keyword tab that was opened. We now opted for an array that keeps track of all open sections, which can be multiple.

## Test instructions

This PR can be tested by following these steps:

* Checkout branch and run yarn install and yarn test (when the tests have passed CR you can assume they are fine, though do check them).
* You could expose the store and action to the window in edit.js, and dispatch (store.dispatch( action() ) and see whether the state was changed correctly.

## Quality assurance


Fixes #9305
